projects/plugins/protect/src/js/components/firewall-page/index.jsx

```diff
@@ -467,15 +467,15 @@ const FirewallPage = () => {
 				<div className={ styles[ 'toggle-section__content' ] }>
 					<div className={ styles[ 'toggle-section__title' ] }>
 						<Text variant="title-medium" mb={ 2 }>
-							{ __( 'Enable automatic rules', 'jetpack-protect' ) }
+							{ __( 'Enable automatic firewall protection', 'jetpack-protect' ) }
 						</Text>
 						{ ! isSmall && hasRequiredPlan && displayUpgradeBadge && (
 							<span className={ styles.badge }>{ __( 'NOW AVAILABLE', 'jetpack-protect' ) }</span>
 						) }
 					</div>
 					<Text>
 						{ __(
-							'Protect your site against untrusted traffic sources with automatic security rules.',
+							'Block untrusted traffic sources by scanning every request made to your site. Jetpack’s advanced security rules are updated in real-time to protect your site from the latest threats.',
```

Comment:
```suggestion
							'Block untrusted traffic sources by scanning every request made to your site. Jetpack’s advanced security rules are automatically kept up-to-date to protect your site from the latest threats.',
```

Per a note from the wise @ArSn peb6dq-1eH-p2#comment-908

Removed "updated in real-time" as that is not accurate (yet!).